lib/helpers/index.ts

```diff
@@ -1,2 +1,3 @@
 import * as binary from './binary';
-export { binary }; 
\ No newline at end of file
+import * as multipart from './multiplex';
+export { binary, multipart };
```

Comment:
shouldn't this be `multiplex` instead of `multipart`?

Reply:
@bjouhier Yes absolutely, I fixed it !